src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,91 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CalculatorTest {
+    private static final double DELTA = 0.00001;
+    private static Calculator calculator;
+
+    @BeforeClass
+    public static void setUp() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    public void additionOk() {
```

Comment:
```suggestion
    public void additionPositiveOk() {
```
same for several other test methods